circle.yml

```diff
@@ -49,7 +49,7 @@ test:
         pwd: $BASE_STABLE
 
   # VET
-    - gvm use stable && go vet ./...:
+    - gvm use stable && test -z "$(go tool vet -printf=false . 2>&1 | grep -v Godeps/_workspace/src/ | tee /dev/stderr)":
```

Comment:
protobuf failures due to their errorf style. I was getting it on my branch, but somehow it isn't happening on master